app/matplotlib.ipynb

```diff
@@ -94,7 +100,12 @@
     "colors = np.random.rand(50)\n",
     "sizes = 100 * np.random.rand(50)\n",
     "\n",
-    "# write your code here"
+    "plt.figure(figsize=(12, 6))\n",
+    "\n",
+    "scatter = plt.scatter(x, y, c=colors, s=sizes)\n",
+    "\n",
+    "colorbar = plt.colorbar(scatter)\n",
+    "colorbar.set_label(\"Color Intensity\")"
```

Comment:
You should add `plt.show()` at the end of this cell to ensure the scatter plot is displayed in the notebook.